parmed/amber/asciicrd.py

```diff
@@ -87,7 +95,8 @@ def positions(self):
 
     def close(self):
         """ Close the open file handler """
-        self.closed or self._file.close()
+        if self._own_handle:
+            self.closed or self._file.close()
         self.closed = True
```

Comment:
so what's your solution here? 

something like this?

```python
     def close(self):
         """ Close the open file handler """
        if self._own_handle:
            self.closed or self._file.close()
            self.closed = True
```

This class is a bit weird to me (vs. others): Related issue: https://github.com/ParmEd/ParmEd/issues/778